### PR TITLE
fix ttl not work when ttl column has default value

### PR DESCRIPTION
**Pull request by critical27**
## What type of PR is this?
- [X] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
close https://github.com/vesoft-inc/nebula/issues/4962

#### Description:
As title, the reason is that we only read the ttl property from RowReader, we need to use latest schema as well for data with older schema version.

## How do you solve it?
`QueryUtils::readValue` already handle this.

## Special notes for your reviewer, ex. impact of this fix, design document, etc:
Add some comments.

## Checklist:
Tests:
- [X] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by panda-sheep — APPROVED** [review]
LGTM

**Comment by pengweisong** [comment]
could refer an issue in the community?

File: src/storage/CommonUtils.cpp [hunk 1]
```diff
@@ -5,15 +5,20 @@
...
 namespace storage {
 
 bool CommonUtils::checkDataExpiredForTTL(const meta::SchemaProviderIf* schema,
                                          RowReader* reader,
                                          const std::string& ttlCol,
                                          int64_t ttlDuration) {
-  auto v = reader->getValueByName(ttlCol);
-  return checkDataExpiredForTTL(schema, v, ttlCol, ttlDuration);
+  auto v = QueryUtils::readValue(reader, ttlCol, schema);
+  if (!v.ok()) {
```
Comment (pengweisong) [inline src/storage/CommonUtils.cpp:18]: need here log the bad status for debug? or the info will be lost.

**Comment by pengweisong** [comment]
> could refer an issue in the community?

I have fixed it.

File: tests/tck/features/ttl/TTL.feature [hunk 2]
```diff
@@ -490,3 +490,38 @@ Feature: TTLTest
     Then the result should be, in any order:
       | age |
     And drop the used space
+
+  @wtf
```
Comment (HarrisChu) [inline tests/tck/features/ttl/TTL.feature:494]: delete the label..

Comment (critical27) [reply tests/tck/features/ttl/TTL.feature:494]: Sorry... My bad

File: src/storage/CommonUtils.cpp (hunk 1 above)
Comment (critical27) [reply src/storage/CommonUtils.cpp:18]: 👌

**Review by panda-sheep — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**